REGRESSION(243836@main): [GStreamer] Broke videos in yelp

mcatanzaro opened the pull request:
#### 6cbee1cfa19be6f4e6bcca15577452c7b062034b
<pre>
REGRESSION(243836@main): [GStreamer] Broke videos in yelp
<a href="https://bugs.webkit.org/show_bug.cgi?id=242156">https://bugs.webkit.org/show_bug.cgi?id=242156</a>

Reviewed by Jer Noble.

243836@main removed a call to frame-&gt;loader().willLoadMediaElementURL(),
which broke video playback in Yelp (the GNOME help viewer, not the
popular website). Put it back.

* Source/WebCore/html/HTMLMediaElement.cpp:
(WebCore::HTMLMediaElement::loadResource):

Canonical link: <a href="https://commits.webkit.org/252746@main">https://commits.webkit.org/252746@main</a>
</pre>

jernoble reviewed (approved):
Apologies! I had thought the Cocoa port was the only implementor of `willLoadMediaElementURL()`. LGTM.

mcatanzaro committed:
REGRESSION(243836@main): [GStreamer] Broke videos in yelp
https://bugs.webkit.org/show_bug.cgi?id=242156

Reviewed by Jer Noble.

243836@main removed a call to frame->loader().willLoadMediaElementURL(),
which broke video playback in Yelp (the GNOME help viewer, not the
popular website). Put it back.

* Source/WebCore/html/HTMLMediaElement.cpp:
(WebCore::HTMLMediaElement::loadResource):

Canonical link: https://commits.webkit.org/252746@main
